draw/src/main/res/values/attrs.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Shouldn't this file be part of to the library?

Reply:
It's already a part of the library, I forgot to remove it.